frontend/src/metabase/components/ConfirmContent.tsx

```diff
@@ -6,7 +6,20 @@ import ModalContent from "metabase/components/ModalContent";
 
 import Button from "metabase/core/components/Button";
 
-const nop = () => {};
+const nop = () => {
+  // Do nothing
```

Comment:
Without comment, ESLint would complain that we shouldn't write an empty function.

Reply:
Should we use `_.noop`?

Reply:
Good idea, so we don't need to declare `noop` everywhere. I didn't think of it much at first since this was a JS -> TS conversion and `nop` was already there. But I think I made another `noop` somewhere, and will remove that as well :)